src/Bicep.LangServer/Handlers/BicepTelemetryHandler.cs

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Bicep.LanguageServer.Telemetry;
+using Newtonsoft.Json.Linq;
+using OmniSharp.Extensions.LanguageServer.Protocol.Client.Capabilities;
+using OmniSharp.Extensions.LanguageServer.Protocol.Models;
+using OmniSharp.Extensions.LanguageServer.Protocol.Workspace;
+
+namespace Bicep.LanguageServer.Handlers
+{
+    // This handler is used to send a request from the client to the server(E.g. command sent as part
+    // of completion item of top level declaration snippet) which in turn triggers a request from server to client,
+    // asking the client to log a telemetry event
+    // Flow of events:
+    // 1. workspace/executeCommand request is sent from the client to the server
+    // 2. The above triggers telemetry/event from server to client
+    public class BicepTelemetryHandler : ExecuteCommandHandlerBase<BicepTelemetryEvent>
```

Comment:
Looks like the base class has been renamed:
https://github.com/OmniSharp/csharp-language-server-protocol/blob/1b6788df2600083c28811913a221ccac7b1d72c9/src/Protocol/Features/Workspace/ExecuteCommandFeature.cs#L118-L141

I still think it's worth using this rather than doing it ourselves, as that way you ensure that the exact same serializer is being used.

Reply:
Updated here:
https://github.com/Azure/bicep/commit/62d6763f06b3d53bb8189b517dab0567a37fd0bd

Thank you!